src/game/logic/object/update/aiupdate.h

```diff
@@ -259,6 +259,8 @@ class AIUpdateInterface : public UpdateModule, public AICommandInterface
     unsigned int Get_Next_Mood_Check_Time() const { return m_nextMoodCheckTime; }
     WhichTurretType Get_Current_Turret() const { return m_currentTurret; }
     Object *Get_Goal_Object() { return Get_State_Machine()->Get_Goal_Object(); }
+    const LocomotorSet *Get_Locomotor_Set() const { return &m_locomotorSet; }

```

Comment:
This certainly returns pointer, not reference?

Reply:
100% sure it returns const pointer.